kpatch-build/kpatch-build

```diff
@@ -674,6 +674,10 @@ usage() {
 	echo "		                        (not recommended)" >&2
 }
 
+if ! command -v gawk &> /dev/null; then
+	die "gawk not installed, please install gawk first"
```

Comment:
Sorry, to be clear, I meant to also remove the "please install gawk first", as that bit is obvious and more importantly it makes the error message consistent with other similar errors.

Reply:
Hi @jpoimboe,
 OK,removed the "please install gawk first",please help check again.
Thanks.